thenewboston_node/business_logic/models/account_state.py

```diff
@@ -74,3 +74,5 @@ def validate_node(self):
 
 
 # TODO(dmu) CRITICAL: Assert all attributes are optional
```

Comment:
@AnonymousDapper This TODO should be removed, since you have just implemented it

Reply:
Done.